frontend/views/components/tabs/TabWrapper.vue

```diff
@@ -53,14 +54,21 @@ export default ({
     defaultTab: String // initial tab name
   },
   data () {
+    const appVersion = process.env.GI_VERSION.split('@')[0]
+    const contractsVersion = process.env.CONTRACTS_VERSION
+
     return {
       activeTab: 0,
       activeComponent: null,
       title: '',
       transitionName: '',
       open: true,
       subNav: [{
-        title: `${L('Version')} ${contractsVersion}`
+        html: L('{span_}App Version: {appVersion}{_span}{br_}{span_}Contracts Version: {contractsVersion}{_span}', {
+          ...LTags('span'),
```

Comment:
Is there a reason for this `span` tag?

I tried changing this to: `App Version: {appVersion}{br_}Contracts Version: {contractsVersion}`

And it worked just fine:

![Screenshot 2024-06-19 at 10 31 53 AM](https://github.com/okTurtles/group-income/assets/138706/51c6a163-a5ef-4916-b5f9-ad8a13af1c6d)